packages/nc-gui-v2/composables/useViewData.ts

```diff
@@ -38,6 +38,8 @@ export function useViewData(
     throw new Error('Table meta is not available')
   }
 
+  const loading = ref(false)
```

Comment:
instead of using `$api` from the global state you can just use `const { api, isLoading } = useApi()` :)
Keeps your composable free of unnecessary refs.

`useApi` also provides an error ref that you can use.